src/components/ENSAddress.tsx

```diff
@@ -15,9 +15,10 @@ function ENSAddressSuspended({
   truncate,
   truncateSize,
 }: ENSAddressProps & { truncate?: boolean; truncateSize: number }) {
-  const { eNSNames } = useSnapshot(GoerliENSStore)
+  const { eNSNames } = useSnapshot(ENSStore.networks[Network.Goerli])
```

Comment:
Maybe let's use `{ensNames}`? It's pretty strange naming here :)

Reply:
All variables start with a small letter. This `eNSNames` property is exported from the library class